.github/workflows/beam_PostCommit_Java_Examples_Dataflow_ARM.yml

```diff
@@ -115,6 +120,9 @@ jobs:
                 -Ppush-containers \
                 -Pdocker-repository-root=us.gcr.io/apache-beam-testing/github-actions \
                 -Pdocker-tag=${{env.TAG}} \
+                -PdisableSpotlessCheck=true \
+                -PdisableCheckStyle=true \
+                -PskipCheckerFramework \
```

Comment:
do we have these flags elsewhere?

Reply:
reference:https://github.com/celeste-zeng/beam/blob/e3d321bc54dca8d258c843566db792cb60967c05/.test-infra/jenkins/job_PostCommit_Java_Examples_Dataflow_V2_Java17.groovy#L47 and here https://github.com/apache/beam/blob/8bd2c97351a4e6f8c6bafd260265d3ca2b7e3673/.github/workflows/beam_PreCommit_Java_Examples_Dataflow_Java11.yml#L105C14-L105C35